.github/workflows/npm-cd.yml

```diff
@@ -139,15 +134,40 @@ jobs:
                   npm_scope: ${{ vars.NPM_SCOPE }}
                   publish: "true"
                   github-token: ${{ secrets.GITHUB_TOKEN }}
+            
+            - name: Del package.json.tmpl
+              shell: bash
+              working-directory: ./node
+              run: |
+                rm package.json.tmpl
+                echo $(ls *json*) 
+
+            - name: Check if RC and set a distribution tag for the package
+              shell: bash
+              run: |
+                  if [[ "${GITHUB_REF:11}" == *"rc"* ]]
+                  then
+                    echo "This is a release candidate"
+                    export npm_tag="next"
+                  else
+                    echo "This is a stable release"
+                    export npm_tag="latest"
+                  fi
+                  echo "NPM_TAG=${npm_tag}" >> $GITHUB_ENV
+
+            - name: Check that the release version dont have typo init
+              if: ${{ github.event_name != 'pull_request' && contains(github.ref, '-') && !contains(github.ref, 'rc') }}
+              run: exit 1
```

Comment:
instead of just existing, raise an explaining message with the exit reason, the passed tag and how it should be fixed (e.g. the right format)